taxonomies/packaging_materials.txt

```diff
@@ -1032,531 +2031,808 @@ xx:85 C/PAP
 
 <en:Multilayer composite
 en:Glass/tinplate, Glass + tinplate
 de:Glas/Weißblech, Glas + Weißblech, Glas/Weissblech, Glas + Weissblech
 hr:staklo/bijeli lim
 xx:97 C/X, 97 C/*, 97 CX, 97 C*
+description:en:Composite material with glass and tin plate (steel coated with tin)
+description:nl:Uit glas en blik (vertind plaatstaal) samengesteld materiaal
 
-<xx:97 C/X
-xx:97 C/GL
+#<xx:97 C/X
+#xx:97 C/GL
+#comment:en:This can not exist, as 98 is for glass with a metal (GL does not add anything). 97 is reserverd for tinplate
 
 <xx:97 C/X
 xx:97 C/FE
+description:en:Composite material with glass and tin plate (steel coated with tin)
+description:nl:Uit glas en blik (vertind plaatstaal) samengesteld materiaal
+#comment:en:This should not exist, as 97 is for glass with a tin plate (but that is in part also Fe)
 
 <en:Multilayer composite
 en:Glass/miscellaneous metals, Glass + miscellaneous metals
 de:Glas/verschiedene Metalle, Glas + verschiedene Metalle
 hr:staklo/raznovrsni metali
 xx:98 C/X, 98 C/*, 98 CX, 98 C*
+description:en:Composite material with glass and metal
+description:nl:Uit glas en metaal samengesteld materiaal
 
-<xx:98 C/X
-xx:98 C/GL
+#<xx:98 C/X
+#xx:98 C/GL
+#comment:en:This can not exist, as 98 is for glass with a metal. So what is GL?
 
 <xx:98 C/X
 xx:98 C/FE
+description:en:Composite material with glass and iron
+description:nl:Uit glas en ijzer samengesteld materiaal
 
 <xx:98 C/X
 xx:98 C/ALU
+description:en:Composite packaging with glass and aluminium (should be 96 C/ALU)
+description:nl:Uit glas en aluminium samengesteld materiaal
 
-en:SIG, SIG Combibloc
-xx:SIG, SIG Combibloc
+#
+# Packaging producers and identifications
+#
+
+en:SIG
+xx:SIG
 wikidata:en:Q1397471
+web:en:https://www.sig.biz/en
+description:en:Producer of packaging
+
+<en:SIG
+en:SIG Combibloc
+xx:SIG Combibloc
+web:en:https://www.sig.biz/en/packaging/packaging-explorer
+description:en:Packaging for fluid foods
 
 <en:Multilayer composite
 en:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
 xx:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
-de:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
-bg:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
 fr:Tetra Pak, Tetra Pak Standard, Brique type Tetra Pak standard
-hr:tetra pak, tetrapak
-nl:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
-pt:Tetra Pak, Tetrapak, Tetra Pack, Tetrapack, Standard Tetra Pak
 packaging_shapes:en: en:brick
 wikidata:en:Q659200
+web:en:https://www.tetrapak.com
+description:en:Producer of packaging for wet (mainly liquid) foods
 
-<en:Tetra Pak
-en:Biobased Tetra Pak
-de:Biobasiertes Tetra Pak, Biobasiertes Tetrapak
-fr:Tetra Pak biosourcé, Brique type Tetra Pak biosourcé
+#<en:Tetra Pak
+#en:Biobased Tetra Pak
+#de:Biobasiertes Tetra Pak, Biobasiertes Tetrapak
+#fr:Tetra Pak biosourcé, Brique type Tetra Pak biosourcé
+#This not a product line
```

Comment:
We have this entry in fr_packaging_materials.csv

Can we skip it the same @stephanegigandet ?

Reply:
if so I have to remove the entry from fr_packaging_materials.csv.

Reply:
We need it for the Eco-Score, we should keep it for now.